packages/cli/src/ui/App.tsx

```diff
@@ -637,6 +638,31 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
     return getAllGeminiMdFilenames();
   }, [settings.merged.contextFileName]);
 
+  useEffect(() => {
+    const prompt = config.getQuestion();
```

Comment:
This is going to get re-invoked on every render. Is there a better way?

Reply:
pulled out prompt outside of the effect block. The whole effect block itself has dependencies so should only run when the dependencies change 👍🏼